common/beerocks/bwl/dummy/mon_wlan_hal_dummy.cpp

```diff
@@ -52,6 +52,8 @@ static mon_wlan_hal_dummy::Data dummy_to_bwl_data(const std::string &opcode)
 {
     if (opcode == "STA-UPDATE-STATS")
         return mon_wlan_hal_dummy::Data::STA_Update_Stats;
+    else if (opcode == "RRM-BEACON-REP-RECEIVED")
+        return mon_wlan_hal_dummy::Data::RRM_Update_Beacon_Measurements;
```

Comment:
We end up using only this DATA. So I think it's not actually needed to make this split into DATA/EVENT. Instead, we can put everything in process_dummy_event().

Reply:
Data path is necessary for 2 reasons:
1. to prevent sleeping and un-synced operation on the test_flows.sh.
2. It is a preperation for the future, for commands that needs to return data immediatly that is not base on events, and we want the ability to change it.

Of course we could handle everything in process_event(), but it will be not such clear.